src/Builder/Expression.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * THIS FILE IS AUTO-GENERATED. ANY CHANGES WILL BE LOST!
```

Comment:
I'll note that GitHub didn't auto-collapse this file. I think it's missing from `.gitattributes`; however, it is excluded in `phpcs.xml.dist`, which tells me those aren't entirely in sync (per my earlier comment).

Reply:
Added